src/components/SearchBar/SearchBar.tsx

```diff
@@ -0,0 +1,63 @@
+// SearchBar.jsx
+import { useState } from 'react';
+
+import { Button } from '../Button/Button';
+
+import styles from './searchbar.module.css';
+
+const SearchBar = () => {
+    const [isOpen, setIsOpen] = useState(false);
+    const [selectedOption, setSelectedOption] = useState('');
+
+    const options = [
+        { value: ' ', label: ' ' },
```

Comment:
what for u need this option? with empty label and value?

Reply:
![Screenshot_1](https://github.com/Allocene/fe-react-2024/assets/120042417/9e8a0b3f-f18c-462f-88d3-9413e13e231e)
Without an empty first one, I can't stylize the dropdown list(

Reply:
ok, i see, but please move this select into separate component